### ISSUE TITLE
Training giving strange results

### ISSUE THREAD

**Issue by calebhemara** [sjmoran/DeepLPF#19]
Hey **Sean, totally am inspired by your work.

I've turned DeepLPF into device-agnostic code, to run on "cpu" (I'm on M1 mac and "mps" is still unreliable). I've been successful in testing images based on your existing checkpoints, however am getting strange results when training on my own data, and I can't figure out what would be giving this "look". I've followed the training data image prep as per your readme file.
input:
![img3](https://user-images.githubusercontent.com/79838938/188122948-67e74f6b-83b2-4980-8775-eafd8a0b3538.jpg)
groundtruth:
![img3](https://user-images.githubusercontent.com/79838938/188122992-1c7c28d8-50da-4d63-9ff2-5dae02efb040.jpg)
test:
![img3_TEST_1_1_PSNR_4 726_SSIM_0 202](https://user-images.githubusercontent.com/79838938/188123163-01b418b2-b0fa-4b8a-869a-b3bafdc6cbcd.jpg)

Any suggestions?

**Comment by sjmoran** [comment]
Hi, thank you for your interest in DeepLPF. My suggestion is to check how you are loading and pre-processing the images for ingestion into DeepLPF. Make sure that the dynamic range is catered for appropriately i.e if 8-bit input images, then you normalise the input by 2^8-1 etc. Check what library you are using to load and how that library works with the image format you are loading, there can be surprises there, depending on the image library and format (e.g. tif). When you do find the issue and solution, please post back here so others can learn from it. Thank you!

**Comment by calebhemara** [comment]
Thanks Sean. Turns out it was something pretty obvious, and it wasn't exactly from my data pre-processing, though, for others-- [this is the best solution for Lightroom exporting for training](https://github.com/nothinglo/Deep-Photo-Enhancer/issues/38#issuecomment-449786636). TLDR; 8 bit TIF in ProPhoto RGB.

My issue was that I had assumed there was a torch.load within the training block for starting training from checkpoint, so my results were just from insufficiently trained parameters. I added this and have much better results now. Thanks again!
